src/breadcrumb/breadcrumb.stories.ts

```diff
@@ -0,0 +1,87 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withNotes } from "@storybook/addon-notes";
+import { action } from "@storybook/addon-actions";
+import { withKnobs, boolean, number } from "@storybook/addon-knobs/angular";
+
+import { BreadcrumbModule, DialogModule } from "../";
+
+storiesOf("Breadcrumb", module)
+	.addDecorator(
+		moduleMetadata({
+			imports: [BreadcrumbModule, DialogModule]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("Basic", () => ({
+		template: `
+		<ibm-breadcrumb [noTrailingSlash]="noTrailingSlash">
+			<ibm-breadcrumb-item href="#">
+				Breadcrumb 1
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item href="#">
+				Breadcrumb 2
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item>
+				Breadcrumb 3
+			</ibm-breadcrumb-item>
+		</ibm-breadcrumb>`,
+		props: {
+			noTrailingSlash: boolean("noTrailingSlash", true)
+		}
+	}))
+	.add("Static Overflow", () => ({
+		template: `
+		<ibm-breadcrumb [noTrailingSlash]="noTrailingSlash">
+			<ibm-breadcrumb-item href="#">
+				Breadcrumb 1
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item>
+				<ibm-overflow-menu>
+					<ibm-overflow-menu-option>
+						<ibm-breadcrumb-item href="#">
+							Breadcrumb 2
+						</ibm-breadcrumb-item>
+					</ibm-overflow-menu-option>
+					<ibm-overflow-menu-option>
+						<ibm-breadcrumb-item href="#">
+							Breadcrumb 3
+						</ibm-breadcrumb-item>
+					</ibm-overflow-menu-option>
+					<ibm-overflow-menu-option>
+						<ibm-breadcrumb-item>
+							Breadcrumb 4
+						</ibm-breadcrumb-item>
+					</ibm-overflow-menu-option>
+					</ibm-overflow-menu>
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item>
+				Breadcrumb 5
+			</ibm-breadcrumb-item>
+		</ibm-breadcrumb>`,
+		props: {
+			noTrailingSlash: boolean("noTrailingSlash", true)
+		}
+	}))
+	.add("Dynamic Overflow", () => ({
```

Comment:
This is the only story we really need and this should be the basic (unless we really want the basic for some reason).

Bonus for providing a knob controlling the number of items (`ngFor` them).
There's a pythonic `range` function available with `import { range } from "../common/utils";` that will help with that. It can be used as `*ngFor="let i of range(itemCount)"`. You'll need to wrap it to use it in the template:
```typescript
range(stop: number, start = 0, step = 1) {
	return range(stop, start, step);
}
```

By controlling the number of items and a threshold (which should default to the number spec supports (3?)), users can experience the full range of possibilities of the component.